### DISCUSSION TITLE
Add Adminer to Laragon menu

### DISCUSSION THREAD

**Discussion by jennydupuy** [leokhoa/laragon#387] [General]
Hi,

I saw that adminer was installed in the new version 6.0 of Laragon, that's cool.
Is there a way to add it to the menu like phpMyAdmin or HeidiSQL?

Thank you a lot.

**Comment by rescenic** [comment]
It's already there by default:
[![laragon-heidisql.png](https://i.postimg.cc/85vTShVK/laragon-heidisql.png)](https://postimg.cc/LYmckg5L)

[![laragon-adminer.png](https://i.postimg.cc/66VMmyvB/laragon-adminer.png)](https://postimg.cc/7b61CP9R)
To add Adminer to menu see this thread:
Tutorial: https://github.com/leokhoa/laragon/discussions/388

**Reply by webdeal** [reply]
And for Laragon 7?

**Reply by AWqxKAWERbXo** [reply]
fyi: 

* Adminer was discontinued.
* AdminerEvo was discontinued, too.
* [AdminerNeo](https://github.com/adminerneo/adminerneo/) might be a successor.

**Reply by rescenic** [reply]
> fyi:
> 
> * Adminer was discontinued.
> * AdminerEvo was discontinued, too.
> * [AdminerNeo](https://github.com/adminerneo/adminerneo/) might be a successor.

Adminer version 5 has just been released.
